markdown/dev/reference/api/part/props/en.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Part.asProps()
+---
+
+This `Part.asProps()` method can be used to return the part as an object that is suitable for inclusion in renderprops. We call this internally, when rendering you only need this when you are building your own render system. 
```

Comment:
nit: suggest to improve clarity on the second sentence:
"We call this internally, when rendering you only need this when you are building your own render system."
What I interpret is this: (though honestly I'm not sure what a 'render system' is)
"This method is used internally and is only required when building your own render system"